haiku-kata-solutions/src/main/java/org/eclipse/collections/haikukata/TextProcessorEC.java

```diff
@@ -60,18 +60,17 @@ public String distinctLetters()
 
     public Triple<CharBag, CharBag, CharSet> duplicatesAndUnique()
     {
-        // TODO: Find all of the alphabetic letters from this.getHaikuAsCharAdapter(), convert them to lowercase
-        // and store them in a MutableCharBag.
-        // Hint: Look at select, collectChar, and toBag
-        MutableCharBag chars = null;
-
-        // TODO: Find all the chars with duplicates
-        // Hint: Find a method on MutableCharBag that returns duplicates
-        CharBag duplicates = null;
-
-        // TODO: Find all the unique chars
-        // Hint: Find a method on MutableCharBag that returns unique chars
-        CharSet unique = null;
+        // Find all of the alphabetic letters, convert them to lowercase
+        MutableCharBag chars = this.getHaikuAsCharAdapter()
+                                .select(Character::isAlphabetic)
+                                .collectChar(Character::toLowerCase)
+                                .toBag();
+
+        // Find all the chars with duplicates
+        CharBag duplicates = chars.selectByOccurences(occurences -> occurences > 1);
```

Comment:
there is a more concise way to do this :) look at the methods in  MutableCharBag.
https://eclipse.dev/collections/javadoc/10.4.0/org/eclipse/collections/api/bag/primitive/MutableCharBag.html